packages/framework/src/Resources/config/services_test.yml

```diff
@@ -162,9 +162,7 @@ services:
     Shopsys\FrameworkBundle\Model\Product\ProductDataFactoryInterface:
         alias: Shopsys\FrameworkBundle\Model\Product\ProductDataFactory
 
-    Shopsys\FrameworkBundle\Model\Product\ProductFacade:
```

Comment:
I think this is still necessary

Reply:
Scheduler is properly autowired even when is nullable